esp/bindings/http/platform/httpservice.cpp

```diff
@@ -297,6 +299,50 @@ void CEspHttpServer::traceRequest(IEspContext* ctx, const char* normalizeMethod)
     span->setSpanAttribute("url.full", full);
 }
 
+// Enumeration of "esp" service methods that are never traced. Add new values as additional
+// requests become relevant.
+enum class UntracedGetMethod
```

Comment:
picky naming: What characteristic do these methods have that means they are not traced?  Is it they are simple file requests?  Something else?  That adjective should be used to describe the method, rather than "Untraced"  - because they are untraced for that reason, not the other way around.